src_python/habitat_sim/agent/agent.py

```diff
@@ -95,6 +95,8 @@ class AgentState:
 class AgentConfiguration:
     height: float = 1.5
     radius: float = 0.1
+    cell_height: float = 0.2
```

Comment:
This has bad code smell. There are a lot of navmesh settings and it would be difficult to keep track of them if we add them all here. Perhaps if we add a dictionary of custom NavMesh settings?

Reply:
Or why don't we just attach a Navmesh settings object here on the extreme case

Reply:
@Skylion007  - refer to conversation [here](https://cvmlp.slack.com/archives/C01QSK5L1NY/p1678910859579859?thread_ts=1677541579.171829&cid=C01QSK5L1NY). TL; DR - these changes are only to support the current iteration of the challenge.